### PR TITLE
Quickstart v2

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: components/Header/Header.jsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ export default function Header() {
                     </Link>
                     <nav id="navbar" className={`${styles.flexContainer} ${styles.navbar}`}>
                             <Link to="/">Home</Link>
-                            <Link to="docs">Documentation</Link>
-                            <Link to="./docs/api_explorer">API Explorer</Link>
+                            <Link to="docs/">Documentation</Link>
+                            <Link to="api-explorer/">API Explorer</Link>
```
Comment (hubert-deriv) [inline components/Header/Header.jsx:26]: Isn't it `/docs/` and `/api-explorer/` instead of `docs/` and `api-explorer/`?

File: package.json [hunk 2]
```diff
@@ -13,8 +13,11 @@
     "@tanstack/react-location-devtools": "^3.4.4",
     "@tanstack/react-location-jsurl": "^3.4.4",
     "@tanstack/react-location-simple-cache": "^3.4.4",
+    "@uiw/react-textarea-code-editor": "^2.0.1",
     "@xstate/react": "^2.0.1",
     "core-js": "^3.21.1",
+    "next-remove-imports": "^1.0.6",
```
Comment (hubert-deriv) [inline package.json:19]: I see a next-js plugin here, but we don't use next-js for our project. I also see this plugin used nowhere. Is there perhaps a reason you wanted to use this plugin? If not, I think you can uninstall this. `npm uninstall next-remove-imports` should do the trick.

File: package.json [hunk 3]
```diff
@@ -13,8 +13,11 @@
     "@tanstack/react-location-devtools": "^3.4.4",
     "@tanstack/react-location-jsurl": "^3.4.4",
     "@tanstack/react-location-simple-cache": "^3.4.4",
+    "@uiw/react-textarea-code-editor": "^2.0.1",
```
Comment (hubert-deriv) [inline package.json:16]: I was wondering the same with this plugin, do you perhaps know what we are using this plugin for? Or will we use it for something?

File: components/Header/Header.jsx (hunk 1 above)
Comment (sanjam-deriv) [reply components/Header/Header.jsx:26]: / in front comes automatically so no need to add it to the link

**Pull request merged**